Cargo.toml

```diff
@@ -51,11 +51,17 @@ rand = "0.6"
 
 [features]
 default = ["softsign"]
-ledger = ["signatory-ledger-tm"]
 softsign = []
 yubihsm = ["signatory-yubihsm/usb"] # USB only for now
 yubihsm-mock = ["yubihsm", "signatory-yubihsm/mockhsm"]
+ledgertm = ["signatory-ledger-tm"]
 
 # Enable integer overflow checks in release builds for security reasons
 [profile.release]
 overflow-checks = true
+
+[patch.crates-io]
```

Comment:
Could you push this into a public repo so that everyone can easily build the public version of the KMS. 

Otherwise this looks good and ready to merge.

Reply:
yes, I had actually done that already :) 
forgot to push that too. Ready now